subprojects.yaml

```diff
@@ -0,0 +1,56 @@
+subprojects:
+- name: Helm Core
+  ownersLink: https://raw.githubusercontent.com/helm/helm/master/OWNERS
+  repos:
+  - https://github.com/helm/helm
+- name: Charts
+  owners:
```

Comment:
Should this link to an OWNERS file as is done with other subprojects?

Reply:
With the charts repo being marked at obsolete but there being a collection of charts subproject repos... what would be the right place for an owners file?